### PR TITLE
refactor(components): migrate colors to helix design system variables 

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Migrate all usage of color constants from the legacy ui_style_constants to the new values taken from
the helix design system.

# Review requests

Smoke test all front-end applications

# Risk assessment
high

### CODE REVIEW

**Review by jerader — COMMENTED** [review]
did a quick audit of pd and it looks fine, a few off colors here and there but i'm not too worried it. Thank you for putting the pr up!

Example of the change in color is the trash bin and waste chute
Screen Shot 2024-01-16 at 2 18 36 PM

**Review by mjhuff — APPROVED** [review]
Should be good with the test fixes! I ran through all the major flows on ODD + desktop and didn't notice anything that obviously wrong (the ODD colors look better for sure)...we may need to rethink our background color when our modals render, but I think following up on this later is fine.

Screenshot 2024-01-16 at 3 35 21 PM


Thanks for solving all our merge conflicts!

File: app/src/atoms/Chip/__tests__/Chip.test.tsx [hunk 1]
```diff
@@ -19,8 +19,10 @@ describe('Chip', () => {
     const [{ getByTestId, getByText, queryByLabelText }] = render(props)
     const chip = getByTestId('Chip_basic')
     const chipText = getByText('mockBasic')
-    expect(chip).toHaveStyle(`background-color: ${COLORS.darkBlack20}`)
-    expect(chipText).toHaveStyle(`color: ${COLORS.darkBlack90}`)
+    expect(chip).toHaveStyle(
+      `background-color: ${COLORS.grey90}${COLORS.opacity20HexCode}`
```
Comment (koji) [inline app/src/atoms/Chip/__tests__/Chip.test.tsx:23]: from the previous code and spreadsheet

```suggestion
      `background-color: ${COLORS.black90}${COLORS.opacity20HexCode}`
```

File: app/src/atoms/Chip/__tests__/Chip.test.tsx [hunk 2]
```diff
@@ -94,11 +96,13 @@ describe('Chip', () => {
     const [{ getByTestId, getByText, getByLabelText }] = render(props)
     const chip = getByTestId('Chip_neutral')
     const chipText = getByText('mockNeutral')
-    expect(chip).toHaveStyle(`background-color: ${COLORS.darkBlack20}`)
+    expect(chip).toHaveStyle(
+      `background-color: ${COLORS.grey90}${COLORS.opacity20HexCode}`
```
Comment (koji) [inline app/src/atoms/Chip/__tests__/Chip.test.tsx:100]: same as the above

File: app/src/atoms/Chip/__tests__/Chip.test.tsx (hunk 1 above)
Comment (koji) [inline app/src/atoms/Chip/__tests__/Chip.test.tsx]: from the previous hex value and spreadsheets

```suggestion
      `background-color: ${COLORS.black90}${COLORS.opacity20HexCode}`
```

File: app/src/atoms/Chip/__tests__/Chip.test.tsx (hunk 2 above)
Comment (koji) [inline app/src/atoms/Chip/__tests__/Chip.test.tsx]: same as the above

**Review by koji — APPROVED** [review]
Thank you for solving conflicts!
I left two comments (they are the same thing).

**Comment by koji** [comment]
> Should be good with the test fixes! I ran through all the major flows on ODD + desktop and didn't notice anything that obviously wrong (the ODD colors look better for sure)...we may need to rethink our background color when our modals render, but I think following up on this later is fine.
> 
> Screenshot 2024-01-16 at 3 35 21 PM
> Thanks for solving all our merge conflicts!

I guess it might be time to update Modal component's background part since the current implementation is kind of breaking Storybook UI.

https://s3-us-west-2.amazonaws.com/opentrons-components/edge/index.html?path=/docs/app-molecules-legacymodal--primary

**Review by brenthagen — APPROVED** [review]
smoke tested desktop/ODD/PD - everything looks good aside from the base fixture background and modal overlay background already mentioned

**Pull request merged**